clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -94,16 +86,12 @@ fn suggest_possible_value_subset() {
     let args = IntoIterator::into_iter(args)
         .map(std::ffi::OsString::from)
         .collect::<Vec<_>>();
-    let comp_type = clap_complete::dynamic::bash::CompType::default();
-    let trailing_space = true;
     let current_dir = None;
 
-    let completions = clap_complete::dynamic::bash::complete(
+    let completions = clap_complete::dynamic::complete(
```

Comment:
This is a one-off fix.  We should resolve this closer to the root and add `cargo test -p clap_complete -F unstable-dynamic` to CI
- https://github.com/clap-rs/clap/blob/master/.github/workflows/ci.yml#L67
- https://github.com/clap-rs/clap/blob/master/.github/workflows/rust-next.yml#L57

Reply:
Should it get a new feature set or be included in `next`?

Reply:
If it works with `next`, let's do that.  Otherwise, I wouldn't worry about it too much and just add the exact command above as another step in CI

Reply:
was able to do it with `next`

Reply:
Looks like we need to just switch to the individual command

Reply:
yeah, the problem is `test-ui` as it doesn't build the workspace. Therefor doesn't know about this feature.

Reply:
I put the command directly in the workflow now